apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -14,10 +15,11 @@
 import retrofit2.Retrofit;
 
 public class SampleApplication extends Application {
-  private static final String BASE_URL = "https://githunt-api.herokuapp.com";
+  // Change localhost to your machine's local IP address when running from a device
+  private static final String BASE_URL = "http://127.0.0.1:8080";
```

Comment:
It would be really nice. I agree that not everyone who wants to run android `apollo-sample` will want to install node + run `frontpage-server` themselves.